physics/unified_ugwp.F90

```diff
@@ -349,6 +353,10 @@ subroutine unified_ugwp_run(me,  master, im,  levs, ntrac, dtp, fhzero, kdt,
     real(kind=kind_phys), dimension(im)       :: sgh30
     real(kind=kind_phys), dimension(im, levs) :: Pdvdt, Pdudt
     real(kind=kind_phys), dimension(im, levs) :: Pdtdt, Pkdis
+
+    ! Variables for sequential updating of winds
```

Comment:
This comment is a little misleading since it looks like u/vwnd1 are used even when ugwp_seq_update is F, they're just set differently depending on that option.

Reply:
I will clarify this comment in the code.